examples/ctypes-factorial/pixi.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "pixidemo2"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["liquidcarbon <akscrps@gmail.com>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
+compile = {cmd = "gcc -shared -o src/factorial.so -fPIC src/factorial.c"}
```

Comment:
We try to include a `start` task in all our maybe it could be nice to add a factorial call but with some pre filled values. We do this to avoid user to have to read the README before they can test it.

Reply:
Why not remove the `engine` argument and make it print both runs in the same output. This makes it easy to get a quick overview.